### ISSUE TITLE
Issue with "appearance (-moz-appearance, -webkit-appearance)": js fiddle accessibility issue

### ISSUE THREAD

**Issue by scottaohara** [mdn/content#9036]
MDN URL: https://developer.mozilla.org/en-US/docs/Web/CSS/appearance

#### What information was incorrect, unhelpful, or incomplete?
The JS fiddle for showing the way to custom style a radio button and checkbox does not appropriately associate the `<label>` elements with each form control.  I realize that is not the primary purpose of the example... but I would request the `<label>`s either wrap the form controls, or be given proper `for` attributes to associate the the controls.  

Developers love copy and pasting, and it'd be great if the JS fiddle didn't lack this fundamental accessibility expectation.
